examples/fpga/nexys4a7ddr/data/pins_nexys4a7ddr.xdc

```diff
@@ -0,0 +1,251 @@
+## Based on https://github.com/Digilent/digilent-xdc/blob/master/Arty-A7-100-Master.xdc
```

Comment:
Could you change this to: https://github.com/Digilent/digilent-xdc/blob/master/Nexys-4-DDR-Master.xdc